src/main/java/com/slicingdice/jslicer/SlicingDice.java

```diff
@@ -407,6 +407,18 @@ public JSONObject countEntityTotal() throws IOException {
     }
 
     /**
+     * Make a total query in Slicing Dice API
+     *
+     * @param tables A JSONObject containing the tables in which the total query will be performed
+     * @return A JSONObject with total query result
+     * @throws IOException
+     */
+    public JSONObject countEntityTotal(final JSONObject tables) throws IOException {
```

Comment:
add a version that receives only a list of Strings in addition to the whole JSONObject